msteams-platform/concepts/device-capabilities/browser-device-permissions.md

```diff
@@ -9,27 +9,40 @@ ms.topic: how-to
 # Device permissions for the browser
 
 > [!NOTE]
-> The change to how device permissions are handled in the browser is currently available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md) only. 
-> This change will be generally available (GA) by February 01, 2022.
+> The latest update on how device permissions are handled in the browser is currently available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md) only. 
+> This update will be generally available (GA) by February 01, 2022.
 
-Applications that require device permissions - such as camera or microphone access - now require users to manually grant consent at a per app level in the web browser. Previously, the browser handled how these permissions were granted, but now these permissions will be handled in Microsoft Teams. This has implications on how you design your application if they require these permissions in the browser.
 
-## Change in behavior
-If your application has declared that it needs device permissions in your [application manifest](native-device-permissions.md), then users will be shown an "app permissions" option where they can enable an app’s device permissions. The "app permissions" option can be found in personal apps, task module dialogs, and tabs in chats, channels or meetings.
+Teams apps that require device permissions, such as camera or microphone access, now require users to manually grant permission at a per app level in the web browser. 
+
+Previously, browser handled how to grant access permissions. Now these permissions are handled in Microsoft Teams. This has implications on how you design your application and if they require these permissions in the browser.
+
+## App permissions in the browser
+If your app needs device permissions as mentioned in your [application manifest](native-device-permissions.md#specify-permissions), **App permissions** option is enabled for app’s device permissions. The **App permissions** option is available in the following capabilities: 
+
+* [Personal apps and task module dialogs](#personal-apps-and-task-module-dialogs)
+* [Chats, channel, or meeting tabs](#chat-channel-or-meeting-tabs)
 
 ### Personal apps and task module dialogs
-The "app permissions" setting can be found in the top-right.
+The **App permissions** option is in the upper-right corner of the page.
+
+
 <img src="../../assets/images/tabs/apppermissions.png" alt="App permissions button" width="800"/>
 
-### Chat, channel or meeting tabs
-The "app permissions" setting can be found in the tab dropdown.
+### Chat, channel, or meeting tabs
+The **App permissions** option is in the dropdown of the tab.
+
+
```

Comment:
Please restructure this better. You do not need to give hyperlinks. Please combine the bullets with the image.